Sources/ConsoleKit/Command/CommandError.swift

```diff
@@ -5,15 +5,51 @@ public struct CommandError: Error, CustomStringConvertible {
 
     /// See `Debuggable`.
     public let reason: String
+    
+    /// The name of the attempted command
+    private let name: String?
```

Comment:
The property name here is a bit confusing. It sounds like it is the name of the error, not the attempted command. I would prefer changing it from `name` to `command`.

Reply:
If you have a name that you like better, feel free to suggest it.